app/components/buttons_component.html.erb

```diff
@@ -1,6 +1,2 @@
-<div class="row justify-content-end">
-  <div class="col-auto">
-    <%= form.submit 'Save as draft', class: 'btn btn-primary', id: 'save-draft-button' %>
-    <%= form.submit submit_button_label, class: 'btn btn-primary' %>
-  </div>
-</div>
+<%= form.submit 'Save as draft', class: 'btn btn-primary', id: 'save-draft-button' %>
```

Comment:
Rather than strip out this HTML, can we add the delete button into the ButtonsComponent?  These seems to appear the same between on both the collection and works edit pages.

Reply:
We could do that, though we'd then end up duplicating the delete link since we'd we still need the separate `DeleteComponent` for the dashboard (same link but styled differently).  This was an attempt to centralize the delete links from any location into a single component.

Reply:
Could `DeleteComponent` be called by the `ButtonComponent`?  The ButtonsComponent was intended to be the full last row of the form with all the buttons on it.

Reply:
Yeah, I could do that, though I'd have do to some logic in the `ButtonComponent` to pick the correct one for the form we are on, since the `DeleteComponent` is specific to a work of a collection.  Let me give it a try.

Reply:
Looks much better now. Thank you.